draco/core/containers.py

```diff
@@ -3340,3 +3340,227 @@ def copy_datasets_filter(
         if isinstance(dest_dset, memh5.MemDatasetDistributed):
             # Redistribute back to the original axis
             dest_dset.redistribute(original_ax_id)
+
+
+    # New container for the estimated bandpass gains and window
+class VisBandpassWindow(FreqContainer):
+    """Container for bandpass gains and their window estimated by running bandpass HyFoReS on hybrid beam-formed visibilities"""
+
+    _axes = ("pol", "freq") #Q8: do I need to declare _axes? What is the point of this? might have to add pol
```

Comment:
You will only have to declare the polarisation axis here, since the frequency axis is derived from `FreqContainer`